src/security/authentication.service.ts

```diff
@@ -77,7 +77,7 @@ export class AuthenticationService {
       this.userProfileApi.get().subscribe(
         (profile: UserProfileModel) => {
           this.profile$.next(profile);
```

Comment:
Why `profile$` and not `profile` ?

Reply:
And explicitly naming `profileObservable` or `profileSubscription` ?

Reply:
Hungarian notation vs Finnish notation, I don't have a preference about that. As you wish ;)

Reply:
I agree with that : https://medium.com/@benlesh/observables-and-finnish-notation-df8356ed1c9b :yum:

Reply:
With TypeScript we don't need a specific notation. The type should give enough information about what is really contained in the var (so if it is typed as `any` ... :cop:).

Reply:
ok, i rename it to "profileSubject".